templates/usr/local/bin/R-update-package.j2

```diff
@@ -2,17 +2,38 @@
 
 argv = commandArgs(trailingOnly = TRUE);
 if (is.null(argv) | length(argv) < 1) {
-  cat("Usage: R-update-package package [lib] [repos]\n");
+  cat("Usage: R-update-package package [type] [lib] [repos]\n");
   q(status = 1);
 }
 
 package = argv[1];
-lib = ifelse(is.na(argv[2]), '{{ r_packages_lib }}', argv[2]);
-repos = ifelse(is.na(argv[3]), '{{ r_packages_repos.rstrip('/') }}', argv[3]);
+type =  ifelse(is.na(argv[2]), '{{ r_packages_type }}', argv[2]);
+lib = ifelse(is.na(argv[3]), '{{ r_packages_lib }}', argv[3]);
+repos = ifelse(is.na(argv[4]), '{{ r_packages_repos.rstrip('/') }}', argv[4]);
 
-if (package %in% old.packages(lib.loc = lib)[, 'Package']) {
+if (package %in% old.packages(lib.loc = lib, repos = repos)[, 'Package'] & type == 'cran') {
   withCallingHandlers({
     update.packages(package, lib, repos, ask = FALSE);
     cat("changed\n");
   }, warning = stop);
 }
+
+if (type == 'bioconductor') {
+  withCallingHandlers({
+    {% if r_version_35 %}
+      BiocManager::install(package, lib = lib, update = FALSE)
```

Comment:
`update = TRUE`?

Reply:
For me `update = TRUE` will update all the "old" packages. I just want to update the specific ones.
Maybe I don't interpret the doc correctly !?

Reply:
> I just want to update the specific ones. Maybe I don't interpret the doc correctly !?

I don't understand where that's done now

Reply:
`BiocManager::install` both install and update the package we give it